test/test_helper.rb

```diff
@@ -167,6 +168,7 @@ class ActionController::TestCase
 
   setup do
     request.env['warden'] = stub(authenticate!: false, authenticated?: false, user: nil)
+    stub_content_register_policies
```

Comment:
It's only stubbing on every controller tests. As with the stubbing in the cucumber features, it needs to be stubbed in any controller test that is rendering an edit form on an edition that can be tagged to policies. I figured it was easy and less brittle to stub a register for all of them rather than including the stubbing just in the tests that render an edition edit form.